### DISCUSSION TITLE
DNNDEV Xmod Pro module

### DISCUSSION THREAD

**Discussion by aloscher** [vanjarosoftware/Vanjaro.Platform#1596] [Q&A]
I am a huge XMod Pro user for development and customization. I would like to know if the XMod Pro module could be installed as an extension on a Vanjaro DNN install?

Great work on this platform. I am hugely impressed and this creates way more power for developers and end-users to build sites quick when the site is less complex. I would love for this to be a solution for my more complex sites as well.

**Comment by Tychodewaard** [comment]
I have had no issues installing DNN extensions. I did not try Xmod but if it is DNN 9.9.1 compatible, it should install. 

Curious to hear more about 'complex sites' as I worked with Vanjaro for over a year now and you can create quite complex things. You 'just' have to find the way the platform expect you to do things.

**Reply by aloscher** [reply]
Complex sites for me are data driven. I usually build dynamic list and detail pages with Xmod like Blogs, News/Event listing, Photo Galleries,  etc... This also allows my clients to use a forms to add, edit, and delete content for simple content editing.

**Reply by Tychodewaard** [reply]
I use plantanapp for custom development and integrate that in Vanajro. You can use those tokens inline in Vanjaro or as blocks. Depending on how much freedom a customer wants/can handle.

**Comment by meetmandeep** [comment]
>  I would like to know if the XMod Pro module could be installed as an extension on a Vanjaro DNN install?

Yes. All DNN modules are compatible with Vanjaro as long as they're DNN 9.9.1+ compatible. 

> I would love for this to be a solution for my more complex sites as well.

Vanjaro is built on top of DNN Platform so you should be able to build anything on Vanjaro that you can with DNN.
